e2e/templates/litmus/reboot-engine.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: litmuschaos.io/v1alpha1
+kind: ChaosEngine
+metadata:
+  name: reboot-engine
```

Comment:
I assume this is for reboot node. If yes, can make it clear like `reboot-node`.

Reply:
renamed.